package.json

```diff
@@ -3,7 +3,8 @@
 	"author": "Stefan Thomas <justmoon@members.fsf.org> (http://www.justmoon.net)",
 	"version": "3.0.2",
 	"description": "Port of jQuery.extend for node.js and the browser",
-	"main": "index",
+	"main": "index.cjs",
+	"module": "index.mjs",
```

Comment:
this is not a standard field, and should not exist in any package. the proper way to distribute ESM is with the "exports" field, and adding that is a breaking change.

Reply:
I was reading this https://2ality.com/2019/10/hybrid-npm-packages.html from Dr. Axel... what do you think to add moth `module` and `export` to have a bit of backwards compatibility with rollup, webpack, etc? 🤔